admin_panel/models.py

```diff
@@ -0,0 +1 @@
+from .exams.model import *  # noqa
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider specific imports over wildcard imports</b></div>
<div id="fix">

Consider using specific imports instead of wildcard imports. This can help improve code readability and avoid potential naming conflicts. You might want to import only the necessary models explicitly.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1 +1,2 @@
 -from .exams.model import *  # noqa
 +from .exams.model import Course, Subject, Question, Topic, Exam, Batch
 +# Import other specific models as needed
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/sleep-404/ExamEvoke/pull/8#issuecomment-2499580485>#1f7607</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged